data/transport missions.txt

```diff
@@ -1867,6 +1872,297 @@ mission "Terraforming 5"
+			`	Nolan outstretches his hand to you. "Nice to meet you, Captain."`
+			choice
+				`	"Nice to meet you too."`
+					goto next
+				`	"I take it you're interested in Amy's ideas."`
+			
+			`	"Absolutely. It's an extraordinary concept, and no offense to you Amy, but it's spectacular that someone of such little experience would imagine such a method of terraforming."`
+			
+			label next
+			`	Amy continues. "Nolan wants to help pursue research of my terraforming techniques. Would you be able to give us a lift to <destination>? I'll pay you <payment>."`
+			choice
+				`	"Sure, I'd be glad to take you there."`
+					accept
+				`	"Sorry, I have other places to be."`
+					decline
+	
+	on complete
+		payment 50000
+		event "terraforming research" 120 180
+		dialog `The whole journey to <planet>, Amy and Nolan discussed the multiple planets within the Dirt Belt that would be perfect candidates for terraforming. "Nolan and I will spend the next few months here researching my theories on terraforming," Amy says as she steps off of your ship. "I'll make sure to contact you once we're ready to try terraforming another planet." She hands you <payment>, then follows Nolan out of the spaceport.`
+
+
+
+event "terraforming research"
+
+mission "Terraforming 8"
+	landing
+	name "Terraforming Research"
+	description "Meet with Amy and Nolan on <destination> to talk about experimentally terraforming another planet."
+	destination "Valhalla"
+	to offer
+		has "event: terraforming research"
+	
+	on offer
+		dialog `You receive a message from Amy: "Hello, Captain. I just wanted to tell you that Nolan and I have recently published our work on 'Reduce Terraforming Expenses Through Equilibrium Mapping,' and we've received a grant from the Deep for it! We're going to begin experiment with terraforming on a different world. Don't worry, the Republic has approved our experiment this time, so we won't get dragged off for any more cases of an 'unlawful use of an asteroid.' If you're interested in helping, pick us up on <destination>."`
+
+
+
+mission "Terraforming 9"
+	landing
+	name "Terraforming Tundra"
+	description "Travel to <destination> with Amy, Nolan, and four others."
+	destination "Tundra"
+	passengers 6
+	cargo "scientific equipment" 10
+	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	to offer
+		has "Terraforming 8: done"
+	
+	on offer
+		conversation
+			`The spaceport is crowded with ships and people when you land. Amy's publication must have stirred up quite the commotion among terraforming scientists.`
+			`	Spaceport security clears a path to your ship for Amy, Nolan, and four others. You notice that a few among the crowd of people are jeering and holding signs, one reading "Don't asteroid my planet!" and a half-eaten sandwich gets thrown in her path as she approaches.`
```

Comment:
Maybe change `four others` to something like `four people you do not recognize`.

Reply:
If the "her" here refers to Amy, or if it is intended to refer to the crowd or to the group, then it should be changed to say so, as otherwise it's ambiguous.